receiver/awscontainerinsightreceiver/internal/host/ec2metadata.go

```diff
@@ -73,6 +75,10 @@ func newEC2Metadata(ctx context.Context, session *session.Session, refreshInterv
 }
 
 func (emd *ec2Metadata) refresh(ctx context.Context) {
+	if emd.localMode {
+		emd.logger.Debug("Running emd in local mode continue without fetching ec2 metadata")
```

Comment:
The error message is confusing. 

Maybe `Running EC2MetadataProvider in local mode.  Skipping EC2 metadata fetch`